### PR TITLE
doc: add minutes for meeting 2021 May 19th

### PR DESCRIPTION
**Pull request by gireeshpunathil**
Fixes: https://github.com/nodejs/diagnostics/issues/477

### CODE REVIEW

**Review by legendecas — APPROVED** [review]

**Review by RafaelGSS — APPROVED** [review]
LGTM.

**Review by Flarna — APPROVED** [review]

File: wg-meetings/2021-05-19.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+* async_hooks: stabilize part of AsyncLocalStorage [#37675](https://github.com/nodejs/node/pull/37675)
+  * Require some doc changes
+  * Michael will volunteer to help do the final edits.
+
+### nodejs/diagnostics
+
+* Identify async_hooks use cases beyond AsyncLocalStorage [#437](https://github.com/nodejs/diagnostics/issues/437)
+  * Next step is to publish a blog post.
+
+* WG also reviewed the Best practice documentation presentation that is slated to be presented in the openjs conference.
```
Comment (Songkeys) [inline wg-meetings/2021-05-19.md:33]: I'm interested in this presentation. Is there a preview or what time is the conference?

Comment (gireeshpunathil) [reply wg-meetings/2021-05-19.md:33]: sent you a mail with the details.

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by gireeshpunathil** [comment]
landed in c80024c9dac67c3f46e335e4ac63fa85831ed0d0

**Pull request closed without merging**